### PR TITLE
[Email Routing] EmailEvent API runtime

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addressed PCX-5524

### CODE REVIEW

File: content/workers/runtime-apis/email-event.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+pcx_content_type: configuration
+title: EmailEvent
+---
+
+# EmailEvent
+
+## Background
+
+An `EmailEvent` is the event type to programmatically process your emails with a Worker. You can reject, forward or drop emails according to the logic you construct in your Worker.
```
Comment (pedrosousa) [inline content/workers/runtime-apis/email-event.md]:
```suggestion
An `EmailEvent` is the event type to programmatically process your emails with a Worker. You can reject, forward, or drop emails according to the logic you construct in your Worker.
```

File: content/workers/runtime-apis/email-event.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+---
+
+## Syntax: Module Worker
```
Comment (pedrosousa) [inline content/workers/runtime-apis/email-event.md:35]: Not sure if there's a right order, but the FetchEvent page has the "Syntax: Service Worker" section before the Module Worker one.

File: content/workers/runtime-apis/email-event.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+---
+
+## Syntax: Service Worker
+
+`EmailEvent` can be handled in Workers functions written using the Service Worker syntax by attaching to the `email` event with `addEventListener`:
+
+```js
+addEventListener("email", (event) => {
+  event.message.forward("your-name@gmail.com");
```
Comment (pedrosousa) [inline content/workers/runtime-apis/email-event.md]:
```suggestion
  event.message.forward("your-name@example.com");
```
That Gmail address probably exists.

File: content/workers/runtime-apis/email-event.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+  - `Envelope From` attribute of the email message.
+
+- `to` {{<type>}}string{{</type>}}
+
+  - `Envelope To` attribute of the email message.
+
+- `headers` {{<type>}}Headers{{</type>}}
+
+  - An [`Headers` object](https://developer.mozilla.org/en-US/docs/Web/API/Headers).
```
Comment (pedrosousa) [inline content/workers/runtime-apis/email-event.md]:
```suggestion
  - A [`Headers` object](https://developer.mozilla.org/en-US/docs/Web/API/Headers).
```

File: content/workers/runtime-apis/email-event.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+  - Size of the email message content.
+
+- {{<code>}}setReject(reason{{<param-type>}}string{{</param-type>}}){{</code>}} {{<type>}}void{{</type>}}
+
+  - Reject this email message by returning a permanent SMTP error back to the connecting client, including the given reason.
+
+- {{<code>}}forward(rcptTo{{<param-type>}}string{{</param-type>}}, headers{{<param-type>}}Headers{{</param-type>}}{{<prop-meta>}}optional{{</prop-meta>}}){{</code>}} {{<type>}}Promise{{</type>}}
+
+  - Forward this email message to a verified destination address of the account. If you want, you can add extra headers to the email message. Only `X-*` headers are allowed.
+  - When the promise resolves, the message is confirmed to be forward to a verified destination address.
```
Comment (pedrosousa) [inline content/workers/runtime-apis/email-event.md]:
```suggestion
  - When the promise resolves, the message is confirmed to be forwarded to a verified destination address.
```

**Review by pedrosousa — APPROVED** [review]

File: content/workers/runtime-apis/email-event.md (hunk 2 above)
Comment (marciocloudflare) [reply content/workers/runtime-apis/email-event.md:35]: changed

**Pull request merged**